src/components/Page.js

```diff
@@ -0,0 +1,38 @@
+const Page = ({ page }) => {
+  const { pageid, title, description, thumbnail, coordinates } = page;
+  const dist = coordinates[0] === undefined ? 0 : coordinates[0].dist;
```

Comment:
good guard against errors! 🙌

Reply:
not asking you to change your conditional to this, but the following is a little safer:

`typeof coordinates[0] === 'undefined'`